### DISCUSSION TITLE
Will there be any Paid Support Options For Vanjaro

### DISCUSSION THREAD

**Discussion by ecotto7** [vanjarosoftware/Vanjaro.Platform#1299] [General]
Just wanted to see if this has been fleshed out yet.  I am already working on migrating sites to RC2 in anticipation of an official release and would be willing to start paying for support now.

**Comment by meetmandeep** [comment]
We will be launching paid support options for Vanjaro along with the official release. If you need any paid assistance prior to that then please feel free to reach out to us. An email with contact information has been sent to you. Thanks
